### ISSUE TITLE
Comments on new examples with diagrams

### ISSUE THREAD

**Issue by bhaugen** [valueflows/valueflows#550]
Overall comment: this is a huge step forward!

Now I pick some nits.

* https://valueflo.ws/examples/ex-resource.html shows one resource with a specification and two without specifications, but in the YAML, they do have classifications. Would be good to explain that on the diagram caption, or even show the links to the classifications in the diagram.
* https://valueflo.ws/examples/ex-exchange.html "An exchange without prior agreement" still shows an agreement in the diagram as well as in the YAML. Some clarification would be good.
* https://valueflo.ws/examples/ex-planning.html#simple-plan-from-recipe
   * Would be nice to explain what a "Delivery fiber" is.
   * Sensorica is spelled wrong in the YAML.
* https://valueflo.ws/examples/ex-planning.html#fulfillment-and-satisfaction hard to parse the diagram where a commitment satisfies an intent, because the shapes point in the other direction. A directed arrow from the satisfies node to the intent might help.
* https://valueflo.ws/examples/ex-planning.html#proposal-for-barter "Intent to provide transfer 3 gallons apple cider" is a really awkward phrase. Removing the word "transfer" would improve the readability.
* https://valueflo.ws/examples/ex-planning.html#proposal-for-work-commitment "Intent to work..." does not communicate what is happening. It is a request for help, or request for somebody else to work. "Intent to work" reads like "I intend to work..."
* https://valueflo.ws/examples/ex-planning.html#proposals-for-price-list Again the word "transfer" in the diagram node titles interferes with readability and I think is not needed.
* https://valueflo.ws/examples/ex-complex.html This is a great example and diagram, but it would be easier to follow if the first node text said "commitment from Alice to transfer  30kg to Bob" and the next one said "commitment from Claudia to Alice for transportation service". And near the bottom of the diagram are two nodes that Bob gets 30kg apples slightly differently. I am sure there is a technical reason for both nodes, but they are confusing to me as a diagram viewer.

**Comment by fosterlynn** [comment]
@bhaugen please re-check, thanks!
